extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/OperationType.scala

```diff
@@ -28,5 +28,5 @@ object OperationType extends Enumeration {
       CREATETABLE_AS_SELECT, CREATEFUNCTION, CREATEVIEW, DESCDATABASE, DESCFUNCTION, DESCTABLE,
       DROPDATABASE, DROPFUNCTION, DROPTABLE, DROPVIEW, EXPLAIN, LOAD, MSCK, QUERY, RELOADFUNCTION,
       SHOWCONF, SHOW_CREATETABLE, SHOWCOLUMNS, SHOWDATABASES, SHOWFUNCTIONS, SHOWPARTITIONS,
-      SHOWTABLES, SHOW_TBLPROPERTIES, SWITCHDATABASE, TRUNCATETABLE = Value
+      SHOWTABLES, SHOW_TBLPROPERTIES, SWITCHDATABASE, TRUNCATETABLE, UPDATE = Value
```

Comment:
Can we delete this？

Reply:
> Can we delete this？

No,here is a must.

Reply:
No, If update is deleted.we will get an error:
<img width="1298" alt="image" src="https://user-images.githubusercontent.com/33471639/223963624-48cdc417-e370-4558-a877-a05d31a963eb.png">

Reply:
you can delete it